components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -72,6 +72,9 @@ public class ServiceProvider implements Serializable {
     @XmlTransient
     private String tenantDomain;
 
+    @XmlTransient
+    private boolean isIllegibleForDefaultClaimMappings;
```

Comment:
Would be good to think of a simple name that conveys the purpose?

Reply:
Also shouldn't this go inside claim configuration model?

Reply:
> 

Removed app level flag